### DISCUSSION TITLE
Dev mode for third-party auth redirect

### DISCUSSION THREAD

**Discussion by paulshen** [supabase/supabase#978] [Feature Requests]
Is it possible to set `redirectTo` of `auth.signIn` to another domain for development? Once your app is deployed, the site URL in auth settings will be set to your production domain but it would be nice if `localhost` redirect still worked to test the loop in development.

```js
client.auth.signIn(
  {provider: 'google'},
  {redirectTo: 'http://localhost:3000'}
)
```

Further, it'd be nice if the Supabase auth site URL setting took a list of domains (similar to Google OAuth).
![image](https://user-images.githubusercontent.com/2266187/112933832-6c0a9a00-90d5-11eb-9788-76f223f830a3.png)

It is possible today to login to production and then manually copy the auth token into a development domain.

**Comment by rickjerrity** [comment]
This is a known issue being worked on, AFAIK. They're trying to add support for multiple domains.

https://github.com/supabase/supabase/discussions/600
